### PR TITLE
change readme file

### PR DESCRIPTION
**Pull request by AlekseyGusar**

### CODE REVIEW

**Comment by artem-trubin** [comment]
Hi! Please check "Files changed" tab and try to resolve the problem. (tip: autoformatter)

**Comment by AlekseyGusar** [comment]
> Hi! Please check "Files changed" tab and try to resolve the problem. (tip: autoformatter)

I have relolved all conflicts

**Comment by artem-trubin** [comment]
Hey, your problem wasn't the conflict. Please pay more attention to comments. Check "Files changed" tab.

File: README.md [hunk 1]
```diff
@@ -36,7 +36,6 @@ My name is Dmytryj.
 <img src="assets/images/mycat.png" alt="cat" style="width: 100%; height: 100%">
 
 
-
```
Comment (suchov) [inline README.md:39]: do you need this line removing?

File: README.md [hunk 2]
```diff
@@ -145,87 +136,96 @@ P.S. Where all postings before March 2018 have gone to?
...
 ![Hahaha](https://i0.wp.com/catcaresolutions.com/wp-content/uploads/2020/12/cute-cat-with-yellow-headband-on.png?fit=1000%2C1500&ssl=1)
 
-
 ## Hi Guys!
+
 My name is Andrew. And I'm really happy to be enrolled to this course. Wish you all luck!
 ![Cool cat](https://i.pinimg.com/736x/44/ab/03/44ab033488f3b73e1c24e2ae59d5106f.jpg)
 
 ## Hi Everyone!
+
```
Comment (suchov) [inline README.md:171]: and adding those one's?

File: README.md [hunk 3]
```diff
@@ -347,54 +356,61 @@ My name is [Oleksii](https://github.com/oleksii-manzik) and my cat is Arnold
...
 [![Kottans-Frontend](https://img.shields.io/badge/%3D%28%5E.%5E%29%3D-frontend-yellow.svg)](https://github.com/kottans/frontend)
 
-
 Hi, kottans! Nice to meet you!
 
 ##### H E L L O!
+
 :smirk_cat:
 
-_________________________________________________
```
Comment (suchov) [inline README.md:390]: ?

**Review by suchov — CHANGES_REQUESTED** [review]

**Pull request closed without merging**